config/prow/autobump-config/prow-job-autobump-config.yaml

```diff
@@ -29,8 +29,13 @@ prefixes:
     repo: "https://github.com/kubernetes/test-infra"
     summarise: false
     consistentImages: false
-  - name: "k8s-staging-test-infra images"
+  - name: "k8s-staging-test-infra GCR images"
     prefix: "gcr.io/k8s-staging-test-infra"
     repo: "https://github.com/kubernetes/test-infra"
     summarise: false
     consistentImages: false
+  - name: "k8s-staging-test-infra AR images"
+    prefix: "us-docker.pkg.dev/k8s-staging-test-infra/images"
```

Comment:
Do we have this declared in https://git.k8s.io/k8s.io ? I don't see any references.

Reply:
Creating that now, this PR can be merged without the infra being created